.github/workflows/conformance-runtime.yaml

```diff
@@ -0,0 +1,422 @@
+
+      - name: Copy Ginkgo binary
+        shell: bash
+        run: |
+          cd test/
+          tar -xf /tmp/.ginkgo-build/test.tgz
+
+      - name: Wait for images to be available
+        timeout-minutes: 10
+        shell: bash
+        run: |
+          for image in cilium-ci operator-generic-ci hubble-relay-ci ; do
+            until docker manifest inspect quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/$image:${{ steps.vars.outputs.sha }} &> /dev/null; do sleep 45s; done
+          done
+
+      - name: Setup runtime
+        timeout-minutes: 10
+        uses: cilium/little-vm-helper@0b7d7157dae56a44dc531c852e7756dc671071af # v0.0.3
+        with:
+          provision: 'false'
+          cmd: |
+            mkdir -p /root/go/src/github.com/cilium/
+            ln -s /host /root/go/src/github.com/cilium/cilium
+            mkdir -p /home/root/go/src/github.com/cilium/
+            ln -s /host /home/root/go/src/github.com/cilium//cilium
+            cp -r /host/test/provision /tmp
+            git config --global --add safe.directory /host
+            export CILIUM_IMAGE=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/cilium-ci:${{ steps.vars.outputs.sha }}
+            export CILIUM_DOCKER_PLUGIN_IMAGE=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/docker-plugin-ci:${{ steps.vars.outputs.sha }}
+            export PROVISION_EXTERNAL_WORKLOAD=false
+            export VMUSER=root
+            echo '127.0.0.1 localhost' >> /etc/hosts
+            echo '::1 localhost' >> /etc/hosts
+            /tmp/provision/runtime_install.sh
+            service docker restart
+
+      - name: Runtime tests
+        if: ${{ matrix.focus == 'f-01-agent' || matrix.focus == 'f-02-datapath' }}
+        timeout-minutes: 20
+        shell: bash
+        run: |
+          cat > test/cilium-ssh-config.txt << EOF
+          Host runtime
+            HostName 127.0.0.1
+            User root
+            Port 2222
+            UserKnownHostsFile /dev/null
+            StrictHostKeyChecking no
+            PasswordAuthentication no
+            LogLevel FATAL
+          EOF
+          cd test
+          export INTEGRATION_TESTS=true
+          ./test.test \
+          --ginkgo.focus="${{ steps.vars.outputs.focus }}" \
+          --ginkgo.skip="${{ matrix.skip }}" \
+          --ginkgo.seed=1679952881 \
+          --ginkgo.v -- \
+          -cilium.provision=false \
+          --cilium.image=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/cilium-ci \
+          -cilium.tag=${{ steps.vars.outputs.sha }}  \
+          --cilium.operator-image=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/operator \
+          -cilium.operator-tag=${{ steps.vars.outputs.sha }} \
+          --cilium.hubble-relay-image=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/hubble-relay-ci \
+          -cilium.hubble-relay-tag=${{ steps.vars.outputs.sha }} \
+          -cilium.operator-suffix=-ci \
+          -cilium.SSHConfig="cat ./cilium-ssh-config.txt"
```

Comment:
nit: Is there a specific pattern to the usage of flags that start with `-` vs. `--` here? They seem inconsistent, but I don't know if this has a functional impact or not.